cm-vault/vault-config/_shared/start_vault_service.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+NODE_PORT=30200
+CONTAINER_PORT=8200
+SERVICE="service/vault"
+NAMESPACE="vault"
+
+echo -en "kubectl port-forward $SERVICE $NODE_PORT:$CONTAINER_PORT -n $NAMESPACE\n disown" > run_service.sh && chmod +x run_service.sh
```

Comment:
Ideally we should actually show if this command errors. I am not sure if we can do something like 'start this process, exit with error if it fails, then send it to background and continue with other things'
(If we cannot that's fine)

Reply:
I tried, but kubectl port-forwarding is executed in the foreground by default. If make it run on the back end, it will not output error messages.

I don't have a good idea to solve this problem. Could you give me some suggestions?

Reply:
I think this could be done in a separate PR if we have more time towards the end of GSoC. I created a separate issue for this https://github.com/cert-manager/testing-addons/issues/17